environments/development/common/cognito.tf

```diff
@@ -1,11 +1,24 @@
 module "cognito" {
```

Comment:
Just in case we reuse this module to create pools for other kinds of flow, I wonder if a better name would be `dev_hub_cognito` or something?

Reply:
Good point, well presented - I'll rename it here.